widgets/src/components/App/Home.js

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
-import { Typography } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+
+import { createWidgetEvent, PDA_CONFIG_ON_UPDATE } from 'custom-elements/customEventsUtils';
+
+setTimeout(() => {
```

Comment:
why do we need to set it to local storage?

Reply:
The Badge config should be saved in a upper level so it can be shared across the solution. As its setup relies on end user defined values, they cannot be on the level of component (each use would require a mapping of values/colors). @werbth and I have discussed some ways of how we will be able to pass the values, but if that will need a fetch (for example for global PDA configs), we have to make sure that badges are updated after the values are fetched.